css/properties.json

```diff
@@ -703,24 +707,24 @@
     "groups": [
       "WebKit Extensions"
     ],
-    "initial": "<code>rgb(0, 0, 0)</code>",
+    "initial": "<code>currentColor</code>",
     "appliesto": "allElements",
-    "computed": "asSpecified",
+    "computed": "'color'",
     "order": "uniqueOrder",
     "status": "nonstandard"
   },
   "-webkit-text-stroke-width": {
     "syntax": "&lt;length&gt;",
     "media": "visual",
     "inherited": true,
-    "animationType": "discrete",
+    "animationType": "no",
     "percentages": "no",
     "groups": [
       "WebKit Extensions"
     ],
     "initial": "<code>0</code>",
     "appliesto": "allElements",
-    "computed": "asSpecified",
+    "computed": "absolute width",
```

Comment:
> Don't know we have such a keyword.

Looks like this is not fixed yet. This keyword doesn't exist yet. You need to add it to css/properties.json and you need to add an entry for it in the translation file l10n/css.json. If you have any questions about that, please let me know.

Sebastian